packages/hardhat/functions/00_backstory_gpt.ts

```diff
@@ -0,0 +1,49 @@
+// [1] ARGUMENT DECLARATION //
+
+// gets: character class for the backstory.
+const CHARACTER_CLASS = args[0]; // --------------> |   Wizard
+
+// gets: character race for the backstory.
+const CHARACTER_RACE = args[1]; // --------------> |   Elf
+
+const CHARACTER_NAME = args[2]; // --------------> |   Eldon
+// [2] PROMPT ENGINEERING //
+
+const prompt = `Generate a D&D 5e backstory for ${CHARACTER_NAME}, a Level 1 ${CHARACTER_CLASS} who is a ${CHARACTER_RACE}. The backstory should be extremely short, dense, powerful, engaging, coherent, and suitable for a fantasy RPG setting. Just reply with the one paragraph backstory and nothing else.`;
+
+// [3] AI DATA REQUEST //
+
+// requests: OpenAI API using Functions
+const openAIRequest = await Functions.makeHttpRequest({
```

Comment:
Is this await required? It seems that await the request twice?

Reply:
let me check

Reply:
yes that is redundant. can probably just assign response = await Functions etc, ty